lib/engine/game/g_1824_cisleithania/meta.rb

```diff
@@ -12,9 +12,8 @@ module Meta
 
         DEPENDS_ON = '1824'
 
-        DEV_STAGE = :prealpha
+        DEV_STAGE = :alpha
 
```

Comment:
> I removed GAME:_IS_VARIANT_OF meta info, as that ment 1824 Cisleithania did not appear in the create game part. As it does not appear as a variant when creating 1824 I thought it was best. Or should it remains and something be changed in 1824 to make it work?

The variant needs `GAME_IS_VARIANT_OF`, and the "parent" game needs the variant added under `GAME_VARIANTS`. So, change this back...

```suggestion

        GAME_IS_VARIANT_OF = G1824::Meta
```

...and in `g_1824/meta.rb` add something like this (most important bit here is that `title` is set correctly, the other fields can be changed how you like):

```ruby
        GAME_VARIANTS = [
          {
            sym: :cis,
            name: 'Cisleithania',
            title: '1824 Cisleithania',
            desc: 'Alternate map for 2-3 players',
          },
        ].freeze
```